python/cudf/dataframe/datetime.py

```diff
@@ -249,6 +249,26 @@ def find_last_value(self, value):
         value = columnops.as_column(value).as_numerical[0]
         return self.as_numerical.find_last_value(value)
 
+    @property
+    def is_unique(self):
+        return self.as_numerical.unique_count() == len(self)
```

Comment:
Just to make it more resilient to implementation changes in the future.
```suggestion
        return self.as_numerical.is_unique
```